Common/Header/LKInterface.h

```diff
@@ -103,6 +103,7 @@ typedef enum{
 	ckResetComm,
 	ckDspMode,
 	ckOtimizerPointsToggle,
+	ckAirspaceLookup,
```

Comment:
if you add CustomKey, you need also add CustomKeyLabel

Reply:
done!